specifyweb/frontend/js_src/css/main.css

```diff
@@ -240,6 +240,10 @@
   optgroup {
     @apply font-bold;
   }
+
+  .text-transform-none {
```

Comment:
you can use the existing tailwind className - [`normal-case`](https://tailwindcss.com/docs/text-transform#:~:text=%3A%20capitalize%3B-,normal%2Dcase,-text%2Dtransform%3A)